Source/WebKit/UIProcess/Cocoa/PlaybackSessionManagerProxy.mm

```diff
@@ -1121,6 +1131,34 @@
     sendToWebProcess(contextId, Messages::PlaybackSessionManager::SetSoundStageSize(contextId.object(), size));
 }
 
+bool PlaybackSessionManagerProxy::prefersAutoDimming() const
+{
+    if (!m_controlsManagerContextId)
+        return [[NSUserDefaults standardUserDefaults] boolForKey:@"WebKitPrefersFullScreenDimming"];
+
+    auto it = m_contextMap.find(*m_controlsManagerContextId);
+    if (it == m_contextMap.end())
+        return [[NSUserDefaults standardUserDefaults] boolForKey:@"WebKitPrefersFullScreenDimming"];
+
+    Ref model = std::get<0>(it->value);
+    return model->prefersAutoDimming();
+}
+
+void PlaybackSessionManagerProxy::setPrefersAutoDimming(bool prefersAutoDimming)
+{
+    [[NSUserDefaults standardUserDefaults] setBool:prefersAutoDimming forKey:@"WebKitPrefersFullScreenDimming"];
```

Comment:
Let's remove this line and rely on `PlaybackSessionModelContext::setPrefersAutoDimming` to do this.